core/trino-main/src/main/java/io/trino/server/ui/PreviewUiQueryRunner.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I think that this file should land in trino-main test sources

Reply:
fixed, thanks